mention-bot.js

```diff
@@ -415,8 +420,8 @@ async function guessOwnersForPullRequest(
 ): Promise<Array<string>> {
   var diff = await fetch(repoURL + '/pull/' + id + '.diff');
   var files = parseDiff(diff);
-  var defaultOwners = getDefaultOwners(files, config.alwaysNotifyForPaths);
-
+  var defaultOwners = getMatchingOwners(files, config.alwaysNotifyForPaths);
+  var fallbackOwners = getMatchingOwners(files, config.fallbackNotifyForPaths);
   if (!config.findPotentialReviewers) {
       return defaultOwners;
```

Comment:
I'm not really sure what that would solve. As if you don't want to find potential users then you could just use the `alwaysNotifyForPaths` instead. Or am I missing something here :point_up: ?
When adding this feature my initial though was to change the config to something like
```json
 "notifyForPaths": [
    {
      "name": "ghuser", 
      "files": ["src/js/**/*.js"],
      "always": true
    }
  ]
```
But that would have broken the API totally for the end user. This is to keep the old behavior and just add another feature. 
We might change the variable names to something like (`defaultOwners => alwaysNotifiedUsers` and `fallbackOwners => fallbackNotifiedUsers`) to make it clear what they are.